src/stemmabench/stemma_generator.py

```diff
@@ -131,6 +166,7 @@ def generate(self):
             level_name += f":{self.depth - remaining_depth - 1}"
             # Decrease remaining depth
             remaining_depth -= 1
+        # Missing manuscript.
```

Comment:
I am not sure what this comment refers to ?

Reply:
- [x] Remove unecessary comment.

I was intending to incorporate an missing manuscript step during the generation process, thereby altering the existing output stemma property (for RAM use). But finally, it didn't serve.